packages/EasyCodingStandard/config/common/docblock.yml

```diff
@@ -31,3 +31,6 @@ services:
 
     # remove variable name from @var and @type annotations
     PhpCsFixer\Fixer\Phpdoc\PhpdocVarWithoutNameFixer: ~
+
+    # remove useless comments such as "Class constructor."
+    SlevomatCodingStandard\Sniffs\Commenting\ForbiddenCommentsSniff: ~
```

Comment:
This is already covered better above by https://github.com/Symplify/Symplify/blob/e320cab883e732fac47c1a95ea6c7ad24c6827f5/packages/EasyCodingStandard/config/common/docblock.yml#L3